javascript/javascript1/week1/age.js

```diff
@@ -0,0 +1,6 @@
+/* Age-ify (A future age calculator) */
+
+let yearOfBirth = 1992;
+let yearFuture = 2032;
+let age = yearFuture - yearOfBirth;
+console.log("You will be " + age +" years old in " + yearFuture ) ;
```

Comment:
Remember you can use [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), which are less error prone and better for your fingers (to avoid these extra " and +) :relieved:

Reply:
Same for `olddog.js`